ddtrace/context.py

```diff
@@ -53,6 +54,7 @@ def __init__(
         meta=None,  # type: Optional[_MetaDictType]
         metrics=None,  # type: Optional[_MetricDictType]
         lock=None,  # type: Optional[threading.RLock]
+        traceparent=None,  # type: Optional[str]
```

Comment:
This has a bit of a smell to it. I don't think we should be having to change the public API to add support for OTel. We do have sufficient support for it already with `_meta`, should we just have an internal API for this state instead? `_set_meta()`?

Reply:
I can just put traceparent into the meta object and instantiate the Context with the meta.